Classes/Tasks/FixMigrationTo1_4Task.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Sng\Additionalscheduler\Tasks;
+
+/*
+ * This file is part of the "additional_scheduler" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+use Sng\Additionalscheduler\BaseEmailTask;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use Sng\Additionalscheduler\Templating;
+use Sng\Additionalscheduler\Utils;
+use TYPO3\CMS\Scheduler\Task\AbstractTask;
+
+class FixMigrationTo1_4Task extends AbstractTask
```

Comment:
I think this must be a Commands instead of a Task, this process is a one shot thing and must not be diplayed each time we add a Task.